sdk/download.go

```diff
@@ -50,15 +52,38 @@ func TarballURL(version string) string {
 	return u.String()
 }
 
-func DownloadFile(file, url string) error {
-	plog.Infof("Downloading %s to %s", url, file)
+func DownloadFile(file, fileURL string, client *http.Client) error {
+	plog.Infof("Downloading %s to %s", fileURL, file)
 
+	// handle bucket urls by using api to get media link
+	parseURL, err := url.Parse(fileURL)
+	if err != nil {
+		return err
+	}
+	if parseURL.Scheme == "gs" {
+		if client == nil {
+			return fmt.Errorf("DownloadFile requires non-nil client with gs:// links")
```

Comment:
Seems to work fine with public gs:// links with the http default client. I can just remove the logic here since client isn't used until `downloadFile` is called which already sets it to DefaultClient when nil.